content/handbook/security/corporate/end-user-services/supported-apps/zoom/zoom-meetings.md

```
@@ -0,0 +1,127 @@
+title: Zoom ミーティング
+description: ミーティングのスケジュールとホスト
+upstream_path: /handbook/security/corporate/end-user-services/supported-apps/zoom/zoom-meetings/
+upstream_sha: 1e195b58b9f249ff10bd0e705106c320fee86141
+translated_at: "2026-05-11T00:00:00Z"
+translator: claude
+stale: false
+---
+
+## 目次
+
+- [Zoom ミーティングのセットアップ](#setting-up-a-zoom-meeting)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)